src/app/tests/suites/certification/Test_TC_WNCV_4_1.yaml

```diff
@@ -0,0 +1,220 @@
+    as DUT
+
+config:
+    cluster: "Window Covering"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+
+    - label: "TH sends DownOrClose command to DUT"
+      command: "DownOrClose"
+
+    - label: "TH waits for 5-15 seconds movement(s) on the DUT"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 5000
+
+    - label:
+          "If (PA_LF & LF) TH reads CurrentPositionLiftPercent100ths attribute
+          from DUT"
+      command: "readAttribute"
+      attribute: "CurrentPositionLiftPercent100ths"
+      PICS: "WNCV_PA_LF && WNCV_LF"
+      response:
+          constraints:
+              type: percent100ths
+              notValue: 0
+              minValue: 1
+              maxValue: 10000
+
+    #Disabled due to issue #14230
+    - label: "TH sends GoToLiftPercentage command with 25 percent to DUT"
+      disabled: true
+      command: "GoToLiftPercentage"
+      arguments:
+          values:
+              - name: "LiftPercentageValue"
+                value: 25
+              - name: "LiftPercent100thsValue"
+                value: 2500
+
+    - label: "Wait 1000ms"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 1000
+
+    - label:
+          "If (PA_LF & LF) TH reads TargetPositionLiftPercent100ths attribute
+          from DUT"
+      disabled: true
+      command: "readAttribute"
+      attribute: "TargetPositionLiftPercent100ths"
+      PICS: "WNCV_PA_LF && WNCV_LF"
+      response:
+          value: 2500
+
+    - label:
+          "If (PA_LF & LF) TH reads TargetPositionLiftPercent100ths attribute
+          from DUT"
+      command: "readAttribute"
+      attribute: "TargetPositionLiftPercent100ths"
+      PICS: "WNCV_PA_LF && WNCV_LF"
+      response:
+          constraints:
+              type: percent100ths
+              minValue: 1
+              maxValue: 10000
+
+    - label: "TH set a timeout of 5-10 minutes for failure"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 5000
+    #Need Bitwise operator support #14491
+    - label: "TH reads OperationalStatus attribute from DUT"
+      command: "readAttribute"
+      attribute: "OperationalStatus"
+      response:
+          constraints:
+              type: map8
+
+    - label:
+          "If (PA_LF & LF) TH reads CurrentPositionLiftPercent100ths attribute
+          from DUT"
+      command: "readAttribute"
+      attribute: "CurrentPositionLiftPercent100ths"
+      PICS: "WNCV_PA_LF && WNCV_LF"
+      response:
+          constraints:
+              type: percent100ths
+              minValue: 1
```

Comment:
Current position value MUST BE 2500 not within a range

Reply:
i will update it